packages/cloud_firestore/README.md

```diff
@@ -60,6 +60,19 @@ class BookList extends StatelessWidget {
 }
 ```
 
+Running a transaction
+
+```dart
+final DocumentReference postRef = Firestore.instance.document('posts/123');
+final TransactionHandler transactionHandler = (Transaction tx) async {
```

Comment:
Assigning functions to variables this way is against the Dart style guide (not sure about Flutter)
Any reason why this is not passed as a closure or at least a nested function?
See also this linter rule http://dart-lang.github.io/linter/lints/prefer_function_declarations_over_variables.html

Reply:
With sound dart or dart 2 on the horizon, the goal is to express the intent of the type, so with function expressions it is good to express the consumed type on the left side. So with that said, it seems reasonable to to express clearly the expressed type, so the reader can clearly read the story with in moments of landing. 
https://www.dartlang.org/guides/language/effective-dart/design#avoid-annotating-types-on-function-expressions

Reply:
In this particular case I think it would be concise and readable to define a closure inside the invocation of `runTransaction()`, like we do for `setState()`. @kroikie, do you agree?